.github/workflows/ci-cd.yml

```diff
@@ -182,7 +182,7 @@ jobs:
 
       - name: Check entrypoint import
         run: |
-          python -c 'import modal._container_entrypoint'
+          python -c 'import modal._container_entrypoint; import modal._runtime.asgi'
```

Comment:
A while back we made this lazily importable so we don't pull in the `aiohttp` stack unless necessary; but it's important to verify that this _can_ work with the container requirements.